src/Distribution/Nixpkgs/Haskell/Stack/PrettyPrinting.hs

```diff
@@ -74,10 +75,15 @@ overrideHaskellPackages oc packages = vcat
     , "};"
     , ""
     ]
-  , "in callPackage <nixpkgs/pkgs/development/haskell-modules> {"
+  , "in callPackage (nixpkgsPath + \"/pkgs/development/haskell-modules\") {"
   , nest 2 $ vcat
     [ attr "ghc" ("pkgs.haskell.compiler." <> toNixGhcVersion (oc ^. ocGhc))
     , attr "compilerConfig" "self: extends pkgOverrides (extends stackageConfig (stackagePackages self))"
+    , "haskellLib = import (nixpkgsPath + \"/pkgs/development/haskell-modules/lib.nix\") {"
+    , nest 2 "inherit pkgs;"
+    , nest 2 "inherit (pkgs) lib;"
+    , "};"
+    , "inherit (stdenv) lib;"
```

Comment:
I believe `lib` attribute should be provided by `callPackage` helper. But I might be wrong.